src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return (days * 40) - 50;
```

Comment:
Also, you are doing the same calculation in every if block `(days * 40) - 50;` This spends processing power and makes code slower. Consider only changing the variable inside the if block, and doing the calculation outside the if block in the end of the code.